Update Global Relay config-settings.rst (#3820)
mattermost/docs: source/administration/config-settings.rst

| This feature's ``config.json`` setting is ``"EmailAddress": ""`` with string input.                                                                                  |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+

> Global SMTP Server Timeout

Author: @isacikgoz, I'm not sure whether my explanation is accurate so please correct as needed. :)

> Global SMTP Server Timeout

Reviewer: ```suggestion
Global Relay SMTP Server Timeout
```

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> The number of milliseconds that can elapse before the connection attempt to the SMTP server is abandoned. The default value is 0 with a maximum setting of 1800 milliseconds.

Reviewer: Actually the default value for this configuration parameter is `1800` seconds, and it shouldn't be `0`. Also, there is no maximum value for this.

Author: Great! Is "milliseconds" in the first sentence correct?

Reviewer: Mmm, It should be `seconds` as well.

> The number of milliseconds that can elapse before the connection attempt to the SMTP server is abandoned. The default value is 0 with a maximum setting of 1800 milliseconds.

Author: ```suggestion
The number of milliseconds that can elapse before the connection attempt to the SMTP server is abandoned. The default value is 1800 seconds. There is no maximum value.
```

Reviewer: I removed the maximum value sentence, there is no need for mentioning it :)

+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+

> | This feature's ``config.json`` setting is ``"SMTPServerTimeout": ""`` with numerical input.                                                                          |

Author: ```suggestion
| This feature's ``config.json`` setting is ``"SMTPServerTimeout": "1800"`` with numerical input.                                                                          |
```

Author: @isacikgoz, is this correct or should it be blank (or 0)?
